src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -733,8 +733,27 @@ void AuctionHouseObject::BuildListOwnerItems(WorldPacket& data, Player* player,
 bool AuctionHouseObject::BuildListAuctionItems(WorldPacket& data, Player* player,
         std::wstring const& wsearchedname, uint32 listfrom, uint8 levelmin, uint8 levelmax, uint8 usable,
         uint32 inventoryType, uint32 itemClass, uint32 itemSubClass, uint32 quality,
-        uint32& count, uint32& totalcount, uint8 /*getAll*/, AuctionSortOrderVector const& sortOrder, Milliseconds searchTimeout)
+        uint32& count, uint32& totalcount, uint8 getAll, AuctionSortOrderVector const& sortOrder, Milliseconds searchTimeout)
```

Comment:
What is the thought process behind it being a uint8 and not a bool? I believe we should change it